### PR TITLE
homework 03

### PR DESCRIPTION
**Pull request by Ildarcho**

### CODE REVIEW

File: homework03/backend/src/main/java/io/github/javaasasecondlanguage/flitter/Context.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+/**
+ * Контекст работы приложения.
+ */
+public class Context {
+    //-------------------------------------------
+    // static section
+    private static Context instance;
+
+    public static Context getInstance() {
```
Comment (Al-p-i) [inline homework03/backend/src/main/java/io/github/javaasasecondlanguage/flitter/Context.java:15]: Non-safe publication here
https://shipilev.net/blog/2014/safe-public-construction/

Comment (Ildarcho) [reply homework03/backend/src/main/java/io/github/javaasasecondlanguage/flitter/Context.java:15]: Fixed

**Comment by Al-p-i** [comment]
2/2

**Comment by Al-p-i** [comment]
2/2

**Pull request closed without merging**